[Android 10] Import / export database file browser broken

<!--
Oh no, a bug! It happens. Thanks for reporting an issue with NewPipe. If this is your first bug report, read the following information before proceeding:

Please note, we only support the latest version of NewPipe. In order to check your app version, open the left drawer and click on "About". If you don't have the latest version, upgrade to it and reproduce the problem before opening the issue. The release page (https://github.com/TeamNewPipe/NewPipe/releases/latest) is where you can get it.

P.S.: Our contribution guidelines might be a nice document to read before you fill out the report :) You can find it at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md

To make it easier for us to help you please enter detailed information in the template we have provided below. If a section isn't relevant, just delete it, though it would be helpful to still provide as much detail as possible.
-->

<!-- The comments between these brackets won't show up in the submitted issue (as you can see in the preview). -->

### Version
<!-- Which version are you using? Hopefully the latest! We just told you that above! -->
0.19.4
Running on an S9+


### Steps to reproduce the bug
<!--
1. Go to '...'
2. Press on '....'
3. Swipe down to '....'
-->
1. Try to import a backup database file

<!-- If you can't cause the bug to show up again reliably (and hence don't have a proper set of steps to give us), please still try to give as many details as possible on how you think you encountered the bug. -->

### Expected behavior
<!-- Tell us what you expect to happen. -->
You can browse to the file and open it.

### Actual behaviour
<!-- Tell us what happens instead. -->
Browsing itself is broken. 
The file browser only shows couple folders, and navigation only goes backwards. I.e. if you click ... it goes to the parent directory, but then you cant navigate back, because that folder doesn't show up. See screenshots for details.
Previous versions of the app (as of yesterday) worked fine, and I didn't do anything to my phone since then. 

### Screenshots/Screen recordings
<!-- If applicable, add screenshots or a screen recording to help explain your problem. GitHub supports uploading them directly in the issue text box. If your file is too big for Github to accept, feel free to paste a link from an image/video hoster here instead. -->

![Screenshot_20200529-124555_NewPipe](https://user-images.githubusercontent.com/39849222/83299115-705ee180-a1aa-11ea-94a1-86dd3d1312de.jpg)
![Screenshot_20200529-124702_NewPipe](https://user-images.githubusercontent.com/39849222/83299180-879dcf00-a1aa-11ea-8c61-4d19d5d94455.jpg)
![Screenshot_20200529-124733_NewPipe](https://user-images.githubusercontent.com/39849222/83299200-95535480-a1aa-11ea-9516-86bc2f1effde.jpg)

^^Each of these folders shows up as empty if you tap them.^^

![Screenshot_20200529-124749_NewPipe](https://user-images.githubusercontent.com/39849222/83299225-a308da00-a1aa-11ea-94ad-6dae4ef43717.jpg)


### Logs
<!-- If your bug includes a crash (where you're shown the Error Report page with a bunch of info), copy it to the clipboard (there is a share button for this), head over to our bug report to markdown converter at https://teamnewpipe.github.io/CrashReportToMarkdown/ and paste it. Copy the converted text (it is MUCH easier to read this way) from there and paste it here: -->
It doesn't crash, just doesn't work, so don't have one.
<!-- That's right, here! -->

Here have some error log when operate export.   

```json
{  "user_action":"ui error",
   "request":"",
   "content_language":"zh_TW_#Hant",
   "service":"none",
   "package":"org.schabi.newpipe",
   "version":"0.19.4",
   "os":"Linux Android 10 - 29",
   "time":"2020-05-30 02:06",
   "exceptions":[
      "java.io.FileNotFoundException: /storage/emulated/0/NewPipeData-20200530_100603.zip: open failed: EACCES (Permission denied)\n\tat libcore.io.IoBridge.open(IoBridge.java:496)\n\tat java.io.FileOutputStream.<init>(FileOutputStream.java:235)\n\tat java.io.FileOutputStream.<init>(FileOutputStream.java:125)\n\tat org.schabi.newpipe.settings.ContentSettingsFragment.exportDatabase(ContentSettingsFragment.java:199)\n\tat org.schabi.newpipe.settings.ContentSettingsFragment.onActivityResult(ContentSettingsFragment.java:181)\n\tat androidx.fragment.app.FragmentActivity.onActivityResult(FragmentActivity.java:170)\n\tat android.app.Activity.dispatchActivityResult(Activity.java:8110)\n\tat android.app.ActivityThread.deliverResults(ActivityThread.java:4860)\n\tat android.app.ActivityThread.handleSendResult(ActivityThread.java:4908)\n\tat android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:51)\n\tat android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)\n\tat android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)\n\tat android.app.ActivityThread$H.handleMessage(ActivityThread.java:2022)\n\tat android.os.Handler.dispatchMessage(Handler.java:107)\n\tat android.os.Looper.loop(Looper.java:359)\n\tat android.app.ActivityThread.main(ActivityThread.java:7418)\n\tat java.lang.reflect.Method.invoke(Native Method)\n\tat com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)\n\tat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)\nCaused by: android.system.ErrnoException: open failed: EACCES (Permission denied)\n\tat libcore.io.Linux.open(Native Method)\n\tat libcore.io.ForwardingOs.open(ForwardingOs.java:167)\n\tat libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)\n\tat libcore.io.ForwardingOs.open(ForwardingOs.java:167)\n\tat android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7317)\n\tat libcore.io.IoBridge.open(IoBridge.java:482)\n\t... 18 more\n"
   
],
   "user_comment":""
}
```
File browser have nothing and click "OK" show this for me.

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -20,6 +20,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:logo="@mipmap/ic_launcher"
+        android:requestLegacyExternalStorage="true"
         android:theme="@style/OpeningTheme"
         tools:ignore="AllowBackup">
         <activity

